### DISCUSSION TITLE
Ready to Merge into 2024   [Pull Requests Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#79136] [Pull Requests]
![Header for the Pull Requests category check-in shows a blue starry background with various planets in the distance. The left has some clouds and an octocat in a Hawaiian shirt holding a GitHub mug. To the right there is a blue and red rocket ship taking off.](https://github.com/community/community/assets/92824518/b7f50bb7-5979-4c6f-92aa-8f3dc5347b99)

As the days of December tick by it feels like we’re in a queue for the new year. As we close out 2023 a huge thank you for those who spent part of their year in Community Discussions and the Pull Requests category 🖤. To celebrate the year’s end, join me in praising our top contributors from the past month and 2023 as well as looking back on some great releases which wouldn’t be possible without your feedback. 

### ⚡Pull Requests Praise: Thanks to these fine folks for keeping our Community awesome ✔️
* @​​LauraS-OCC: Shared their team’s [workaround for the Review Changes button not working](https://github.com/orgs/community/discussions/78680) and it is appreciated along with everyone who reported this bug. 
* @FredrikLundkvist: Gave [some great feedback](https://github.com/orgs/community/discussions/78659) in their post and thoroughly explained their use case - thanks for helping GitHub improve! 

### 🏆2023’s Top Contributors: These folks contributed the most in our category for the year, thank you and looking forward to another great 2024!
Top Contributor: @purajit
Second place: @ACwest
Third place: @lobsterkatie

![Three astronauts on an Olympic style podium depicting first, second, and third place against a space background. The astronauts faces are the profile pictures of the top contributors, from left to right: @lobsterkatie(3rd place), @purajit(1st place), @ACwest(2nd place).](https://github.com/community/community/assets/92824518/cd3162ff-108c-434b-ac94-5bd18acb7db2)

### 🎶 Pull Requests' Greatest Hits: Top releases from the past year that leveled up the developer experience ⬆️
* **Pull request merge queue**: Merge queue helps increase velocity in software delivery by automating pull request merges into your busiest branches. Haven’t tried this feature yet? Watch @chrisreddington [demonstrate how they can help you ship faster with less friction](https://www.youtube.com/watch?v=XEZMgohmtts). 
* **Commenting on files in a pull request**: Commenting on files (including deleted, binary, and renamed files) in a pull request became generally available on the web and GitHub Mobile back in April! [Learn more about commenting on a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request).
* **API for reverting a pull request**: Simple, but welcome release that kicked off Pull Requests’ releases for 2023. [View the changelog](https://github.blog/changelog/2023-01-27-api-for-reverting-a-pull-request/). 

Have a favorite release or solve from the past year? Shout it out in the comments! Thanks again for another great year and looking forward to what we learn and accomplish in 2024.

**Comment by georgetoloraia** [comment]
This is a test answer

**Comment by queenofcorgis** [comment]
First check-in of the year has merged in to the Pull Requests category, [read the check-in](https://github.com/orgs/community/discussions/102050).
